Cargo.toml

```diff
@@ -2,6 +2,7 @@
 members = [
   "kindelia",
   "kindelia_core",
+  "kindelia_ws_events"
```

Comment:
Could you change the name to `kindelia_ws`?

Reply:
Done